src/reducers/errors.js

```diff
@@ -0,0 +1,38 @@
+import omit from 'lodash.omit';
+
+import * as actionTypes from '../constants/action-types';
+
+const initialState = {};
+
+const queries = (state = initialState, action) => {
+    switch (action.type) {
+        case actionTypes.RESET: {
+            return {};
+        }
+        case actionTypes.MUTATE_START:
+        case actionTypes.REQUEST_START: {
+            const { queryKey } = action;
+
+            return omit(state, queryKey);
```

Comment:
my reasoning for putting it here is because it will keep it in-sync with the queries state where we reset the query state on MUTATE_START/REQUEST_START